### PR TITLE
updated android logo in README.md

### PR DESCRIPTION
**Pull request by rutuja1121**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ Here's how to generate a [Personal Access Token](https://docs.github.com/en/auth
 
 ### Tech Stack
 
-[![android](https://www.vectorlogo.zone/util/preview.html?image=/logos/android/android-tile.svg)](https://developer.android.com)
+[![android](https://www.vectorlogo.zone/logos/android/android-ar21.svg)](https://developer.android.com)
```
Comment (Anuragtech02) [inline README.md:118]: Can you please use any other logo? This one doesn't look good in dark mode!

**Review by Anuragtech02 — CHANGES_REQUESTED** [review]
Hey, thanks for taking the time to make a PR! Just one correction and we'll be good to go.

**Comment by rutuja1121** [comment]
Yes sure!

**Review by Anuragtech02 — APPROVED** [review]
LGTM! Thanks for the PR ;)

**Pull request merged**